### PR TITLE
Update a releasenotes description

### PR DESCRIPTION
**Pull request by hanxiaop**
**Please provide a description of this PR:**
Address a comment in https://github.com/istio/istio/pull/45577#discussion_r1245599651

### CODE REVIEW

**Comment by hanxiaop** [comment]
It looks like there's no listed reviewer for only releasenotes change. @hzxuzhonghu Can you approve this one? Thanks

File: releasenotes/notes/45577.yaml [hunk 1]
```diff
@@ -3,4 +3,4 @@ kind: bug-fix
 area: istioctl
 releaseNotes:
 - |
-  **Fixed** xDS `proxy-status` has inaccurate Istio version. Note: This fix is optimized for istioctl 1.19 and istiod 1.19. For full functionality, ensure that both istioctl and istiod are upgraded to version 1.19.
+  **Fixed** xDS `proxy-status` has inaccurate Istio version. Note: This fix is optimized for istioctl 1.19 and istiod 1.19. For full functionality, ensure that both istioctl and istiod are upgraded to version 1.19 or later.
```
Comment (hzxuzhonghu) [inline releasenotes/notes/45577.yaml:6]: We donot usually mention it is for which version. We may backport it,  So would suggest removing 1.19 totally.

Comment (hanxiaop) [reply releasenotes/notes/45577.yaml]: Thanks, just revised. Could you please check if the revised one is better?

**Review by hzxuzhonghu — APPROVED** [review]
Thanks

**Pull request merged**